app/assets/schemas/archive-format.json

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
I wonder if we want to have the schemas inside the assets folder, since it's technically not a frontend asset. Do you have a better idea?